tests/flow/test_data/redis_scripts.py

```diff
@@ -72,3 +76,19 @@ def test_set_key(key:str, value:str):
 
 def test_del_key(key:str):
     redis.execute("DEL", [key])
+
+
+def test_model_execute(keys:List[str]):
+    a = torch.tensor([[2.0, 3.0], [2.0, 3.0]])
+    b = torch.tensor([[2.0, 3.0], [2.0, 3.0]])
+    return redisAI.model_execute(keys[0], [a, b], 1)  # assume keys[0] is the model key name saved in redis
+
+
+def test_model_execute_onnx(keys:List[str]):
```

Comment:
def test_model_execute_onnx(tensors: List[Tensor], keys: List[Str], args: List[Str]):

Reply:
I prefer changing it when we change all the script tests, so I won't have to send dummy `LIST_INPUTS' everywhere...